x-pack/legacy/plugins/spaces/public/copy_saved_objects_to_space/components/copy_to_space_flyout.test.tsx

```diff
@@ -17,7 +17,7 @@ import { act } from '@testing-library/react';
 import { ProcessingCopyToSpace } from './processing_copy_to_space';
 import { spacesManagerMock } from '../../spaces_manager/mocks';
 import { SpacesManager } from '../../spaces_manager';
-import { ToastNotifications } from 'ui/notify/toasts/toast_notifications';
+import { ToastNotifications } from '../../../../../../../src/plugins/kibana_legacy/public';
```

Comment:
ToastNotifications shouldn't be used anymore, instead the plugins should consume the core apis directly. For these usages can we leave them on the `ui/` import that just re-exports the `kibana_legacy` one and add a warning comment in there to switch to the core apis instead? Then it stands out and will hopefully caught in the migration process.

Reply:
will do 👍

Reply:
done